atulkrishnathakur_fastcrud_router_api_file_route_71.py

```
@@ -0,0 +1,38 @@
+from typing import Annotated
+from sqlalchemy.orm import Session
+from database.session import get_db
+from database.model_functions.user import (read_all_user,saveUser,saveOrUpdateUser,
+updateUser,deleteUser,readbyoperators)
+import logging
+from fastapi.staticfiles import StaticFiles
+
+router = APIRouter()
+
+@router.post("/upload-file",name="uploadfile")
+def uploadFile(file: UploadFile = File(...)):
```

Ruff (0.13.1)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)